frontend/i18next-scanner.config.js

```
@@ -0,0 +1,55 @@
+      i18nKey: 'i18nKey',
+      defaultsKey: 'defaults',
+      extensions: [''],
+      fallbackKey: false,
+      supportBasicHtmlNodes: true,
+      keepBasicHtmlNodesFor: ['br', 'strong', 'i', 'em', 'span']
+    },
+    lngs: ['en', 'es', 'zh'],
+    fallbackLng: 'en',
+    ns: ['translation'],
+    defaultNs: 'translation',
+    keySeparator: '.',
```

Biome (2.4.6)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)